src/main/scala/karazin/scala/users/group/week2/homework/Homework.scala

```diff
@@ -34,19 +34,34 @@ object Homework:
       !(this < that)
 
     @targetName("addition")
-    infix def +(that: Rational): Rational = ???
+    infix def +(that: Rational): Rational =

```

Comment:
Don't you have `gcd` applied in constructor of `Rational`?
Then you don't need to that here and repeat it everywhere.